src/google-maps/map-marker/map-marker.ts

```diff
@@ -34,42 +33,38 @@ export const DEFAULT_MARKER_OPTIONS = {
   selector: 'map-marker',
   exportAs: 'mapMarker',
 })
-export class MapMarker implements OnInit, OnDestroy, MapAnchorPoint {
+export class MapMarker implements OnInit, OnChanges, OnDestroy, MapAnchorPoint {
   private _eventManager = new MapEventManager(this._ngZone);
-  private readonly _options =
-      new BehaviorSubject<google.maps.MarkerOptions>(DEFAULT_MARKER_OPTIONS);
-  private readonly _title = new BehaviorSubject<string|undefined>(undefined);
-  private readonly _position =
-      new BehaviorSubject<google.maps.LatLngLiteral|google.maps.LatLng|undefined>(undefined);
-  private readonly _label =
-      new BehaviorSubject<string|google.maps.MarkerLabel|undefined>(undefined);
-  private readonly _clickable = new BehaviorSubject<boolean|undefined>(undefined);
-  private readonly _destroy = new Subject<void>();
-
-  @Input()
-  set options(options: google.maps.MarkerOptions) {
-    this._options.next(options || DEFAULT_MARKER_OPTIONS);
-  }
 
-  @Input()
-  set title(title: string) {
-    this._title.next(title);
-  }
+  /**
+   * Title of the marker.
+   * See: developers.google.com/maps/documentation/javascript/reference/marker#MarkerOptions.title
+   */
+  @Input() title: string;
 
-  @Input()
-  set position(position: google.maps.LatLngLiteral|google.maps.LatLng) {
-    this._position.next(position);
-  }
+  /**
+   * Title of the marker. See:
+   * developers.google.com/maps/documentation/javascript/reference/marker#MarkerOptions.position
+   */
+  @Input() position: google.maps.LatLngLiteral|google.maps.LatLng;
 
-  @Input()
-  set label(label: string|google.maps.MarkerLabel) {
-    this._label.next(label);
-  }
+  /**
+   * Label for the marker.
+   * See: developers.google.com/maps/documentation/javascript/reference/marker#MarkerOptions.label
+   */
+  @Input() label: string|google.maps.MarkerLabel;
 
-  @Input()
-  set clickable(clickable: boolean) {
-    this._clickable.next(clickable);
-  }
+  /**
+   * Whether the marker is clickable. See:
+   * developers.google.com/maps/documentation/javascript/reference/marker#MarkerOptions.clickable
+   */
+  @Input() clickable: boolean;
+
+  /**
+   * Options used to configure the marker.
+   * See: developers.google.com/maps/documentation/javascript/reference/marker#MarkerOptions
+   */
+  @Input() options: google.maps.MarkerOptions;
```

Comment:
```suggestion
  @Input() options: Readonly<google.maps.MarkerOptions>;
```
We could enforce that the whole object needs to be set

Reply:
Makes sense, this might break some users who are using this incorrectly (or just not carefully) though.